eclipse-scout-core/src/App.ts

```diff
@@ -385,10 +392,7 @@ export class App extends EventEmitter {
   }
 
   protected _initVersion(options: AppModel) {
-    this.version = scout.nvl(
-      this.version,
-      options.version,
-      $('scout-version').data('value'));
+    this.version = scout.nvl(this.version, options.version, config.get('scout.application.version'));
```

Comment:
I don't think the version tag in the html can be replaced -> we need to discuss it